consensus/istanbul/core/testbackend_test.go

```diff
@@ -17,13 +17,17 @@
 package core
 
 import (
+	bytes2 "bytes"
```

Comment:
Recommend changing this to use the default import name
```suggestion
	"bytes"
```